### PR TITLE
Document new "send to" interactions in log expl.

### PR DESCRIPTION
**Pull request by pcarioufr**
### Preview link

* https://docs-staging.datadoghq.com/pcarioufr/logs_send_to/logs/explorer/?tab=logsearch#send-views-to

### CODE REVIEW

File: content/en/logs/explorer/_index.md [hunk 1]
```diff
@@ -51,11 +51,53 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
-### Saved views
+### Save Views
+
+Use Saved Views to automatically configure your Log Explorer with a preselected set of facets, measures, searches, time ranges, and visualizations. Check the dedicated [saved views documentation][2] to learn more.
+
+### Send Views to
+
+Export your current Log Visualization with the *Send To...* functionality:
+
+{{< img src="logs/explorer/send_view_to.png" alt="Send view to" responsive="true" style="width:30%;>}}
```
Comment (l0k0ms) [inline content/en/logs/explorer/_index.md]:
```suggestion
{{< img src="logs/explorer/send_view_to.png" alt="Send view to" responsive="true" style="width:30%;">}}
```

File: content/en/logs/explorer/_index.md [hunk 2]
```diff
@@ -51,11 +51,49 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+
+Use Saved Views to automatically configure your Log Explorer with a preselected set of facets, measures, searches, time ranges, and visualizations. Check the dedicated [saved views documentation][2] to learn more.
+
+### Send Views to
+
+Export your current Log Visualization with the *Send To...* functionality:
+
+{{< img src="logs/explorer/send_view_to.png" alt="Send view to" responsive="true" style="width:60%;">}}
+
+The *Send To...* button is the go-to for interactions to sent a log explorer view in other contexts, inside Datadog or outside Datadog. Available interactions vary depending on explorer page.
```
Comment (l0k0ms) [inline content/en/logs/explorer/_index.md]:
```suggestion
The *Send To...* button allows you to send your current log explorer view to another page within Datadog or outside of Datadog:
```

File: content/en/logs/explorer/_index.md [hunk 3]
```diff
@@ -51,11 +51,49 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+
+
+{{< tabs >}}
+{{% tab "Log Search" %}}
+
+| Button            | Description                                                                                                          |
+|-------------------|----------------------------------------------------------------------------------------------------------------------|
+| Export to Monitor | Export the query applied to your Logstream in order to create the log monitor query for a new [log monitor][1]       |
```
Comment (l0k0ms) [inline content/en/logs/explorer/_index.md]:
```suggestion
| Export to Monitor | Export the query applied to your Logstream to create the log monitor query for a new [log monitor][1]       |
```

File: content/en/logs/explorer/_index.md [hunk 4]
```diff
@@ -51,11 +51,49 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+| Export to CSV     | Export your current Logstream view with its selected column into a CSV file. You can export up to 5000 logs at once. |
```
Comment (l0k0ms) [inline content/en/logs/explorer/_index.md]:
```suggestion
| Export to CSV     | Export your current Logstream view with its selected columns into a CSV file. You can export up to 5000 logs at once. |
```

File: content/en/logs/explorer/_index.md [hunk 5]
```diff
@@ -51,11 +51,49 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+| Share View     | Share a link to the view with teammates through email, Slack and more (see our [notification integrations][2]) to teammates using the Share View item. |
```
Comment (l0k0ms) [inline content/en/logs/explorer/_index.md]:
```suggestion
| Share View     | Share a link to the current view with your teammates through email, Slack and more. See all [Datadog notification integrations][2] available. |
```

File: content/en/logs/explorer/_index.md [hunk 6]
```diff
@@ -51,11 +51,49 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+
+
+[1]: /monitors/monitor_types/log
+[2]: /integrations/#cat-notification
+{{% /tab %}}
+{{% tab "Log Analytics" %}}
+
+| Button              | Description                                                                                                                                                   |
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Export to Monitor   | Export the query applied to your Log Analytics in order to create the log monitor query for a new [log monitor][1] |
```
Comment (l0k0ms) [inline content/en/logs/explorer/_index.md]:
```suggestion
| Export to Monitor   | Export the query applied to your Log Analytics to create the log monitor query for a new [log monitor][1]. |
```

File: content/en/logs/explorer/_index.md [hunk 7]
```diff
@@ -51,11 +51,49 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+| Export to Dashboard | to embed the corresponding content in a widget in a [dashboard][2]. |
```
Comment (l0k0ms) [inline content/en/logs/explorer/_index.md]:
```suggestion
| Export to Dashboard | Export the current analytic as a widget within a chosen [dashboard][2]. |
```

File: content/en/logs/explorer/_index.md [hunk 8]
```diff
@@ -51,11 +51,49 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+| Generate new Metric | to [generate a metric][3] out of the current query |
```
Comment (l0k0ms) [inline content/en/logs/explorer/_index.md]:
```suggestion
| Generate new Metric | [Generate a new metric][3] out of the current analytic query.  |
```

**Comment by l0k0ms** [comment]
@pcarioufr i think you might as well update the overview of this page in order to change the wording and link of: `Export the content of your Log explorer view to a monitor, a dashboard, or into a CSV file.`

File: content/en/logs/explorer/_index.md (hunk 7 above)
Comment (pcarioufr) [reply content/en/logs/explorer/_index.md]: It can be a new dashboard as well

Comment (l0k0ms) [reply content/en/logs/explorer/_index.md]: suggestion updated

File: content/en/logs/explorer/_index.md [hunk 9]
```diff
@@ -28,7 +28,8 @@ In this view you can:
 * [Build a context to explore your logs](#context).
 * [Visualize your logs as a filtered Logstream or Log Analytics](#visualization).
 * [Setup your log explorer view by creating facets and measure from your logs](#setup).
-* [Export the content of your Log explorer view to a monitor, a dashboard, or into a CSV file.](#export)
+* [Share the content of your explorer view to another page within Datadog or outside of Datadog.](#share-view)
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
* [Share the content of your explorer view to another page within or outside of Datadog.](#share-views)
```

File: content/en/logs/explorer/_index.md [hunk 10]
```diff
@@ -51,11 +52,47 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
 
 {{< img src="logs/explorer/search_your_logs.mp4" alt="Search your logs" video="true" responsive="true" >}}
 
-### Saved views
+### Saved Views
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
### Saved views
```

File: content/en/logs/explorer/_index.md [hunk 11]
```diff
@@ -51,11 +52,47 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+
+Use Saved Views to automatically configure your Log Explorer with a preselected set of facets, measures, searches, time ranges, and visualizations. Check the dedicated [saved views documentation][2] to learn more.
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
Use saved views to automatically configure your log explorer with a preselected set of facets, measures, searches, time ranges, and visualizations. Check the dedicated [saved views documentation][2] to learn more.
```

File: content/en/logs/explorer/_index.md [hunk 12]
```diff
@@ -51,11 +52,47 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+
+### Share Views
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
### Share views
```

File: content/en/logs/explorer/_index.md [hunk 13]
```diff
@@ -51,11 +52,47 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+
+Export your current Log Visualization with the *Share* functionality:
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
Export your current log visualization with the *share* functionality:
```

File: content/en/logs/explorer/_index.md [hunk 14]
```diff
@@ -51,11 +52,47 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+
+{{< img src="logs/explorer/send_view_to.png" alt="Send view to" responsive="true" style="width:60%;">}}
+
+The *Share* button allows you to send your current log explorer view to another page within Datadog or outside of Datadog:
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
Use the *share* button to send your current log explorer view to a CSV file, team member, or create a monitor:
```

File: content/en/logs/explorer/_index.md [hunk 15]
```diff
@@ -51,11 +52,47 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+
+{{< tabs >}}
+{{% tab "Log Search" %}}
+
+| Button            | Description                                                                                                          |
+|-------------------|----------------------------------------------------------------------------------------------------------------------|
+| Export to Monitor | Export the query applied to your Logstream to create the log monitor query for a new [log monitor][1]       |
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
| Export to Monitor | Export the query applied to your logstream to create the query for a new [log monitor][1].       |
```

File: content/en/logs/explorer/_index.md [hunk 16]
```diff
@@ -51,11 +52,47 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+| Export to CSV     | Export your current Logstream view with its selected columns into a CSV file. You can export up to 5000 logs at once. |
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
| Export to CSV     | Export your current logstream view with its selected columns to a CSV file. You can export up to 5,000 logs at once. |
```

File: content/en/logs/explorer/_index.md [hunk 17]
```diff
@@ -51,11 +52,47 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+| Share View     | Share a link to the current view with your teammates through email, Slack and more. See all [Datadog notification integrations][2] available. |
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
| Share View     | Share a link to the current view with your teammates through Email, Slack, and more. See all [Datadog notification integrations][2] available. |
```

File: content/en/logs/explorer/_index.md [hunk 18]
```diff
@@ -51,11 +52,47 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
 
-Use Saved Views to automatically configure your Log Explorer with a preselected set of facets, measures, searches, time ranges, and visualizations.
+[1]: /monitors/monitor_types/log
+[2]: /integrations/#cat-notification
+{{% /tab %}}
+{{% tab "Log Analytics" %}}
+
+| Button              | Description                                                                                                                                                   |
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Export to Monitor   | Export the query applied to your Log Analytics to create the log monitor query for a new [log monitor][1]. |
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
| Export to Monitor   | Export the query applied to your log analytics to create the query for a new [log monitor][1]. |
```

File: content/en/logs/explorer/_index.md [hunk 19]
```diff
@@ -51,11 +52,47 @@ Follow the [guide to search your logs][1] for a detailed explanation of all the
...
+| Export to Dashboard | Export the current analytic as a widget into an existing or a new [dashboard][2]. |
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
| Export to Dashboard | Export the current analytic as a widget to an existing or new [dashboard][2]. |
```

File: content/en/logs/explorer/_index.md [hunk 20]
```diff
@@ -104,7 +141,11 @@ Interact with the upper reserved attributes section:
 
 The **View in context** button updates the search request in order to show you the log lines dated just before and after a selected log—even if they don't match your filter. This context is different according to the situation, as Datadog uses the `Hostname`, `Service`, `filename`, and `container_id` attributes, along with tags, in order find the appropriate context for your logs.
 
-Copy the JSON log content to the clipboard through the **Export** button or keyboard interaction (Ctrl+C/Cmd+C).
+ 
+The **Share** Button is the go-to to share the log opened in side panel to other contexts.
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
Use the **Share** button to share the log opened in side panel to other contexts. 
```

File: content/en/logs/explorer/_index.md [hunk 21]
```diff
@@ -104,7 +141,11 @@ Interact with the upper reserved attributes section:
 
 The **View in context** button updates the search request in order to show you the log lines dated just before and after a selected log—even if they don't match your filter. This context is different according to the situation, as Datadog uses the `Hostname`, `Service`, `filename`, and `container_id` attributes, along with tags, in order find the appropriate context for your logs.
 
-Copy the JSON log content to the clipboard through the **Export** button or keyboard interaction (Ctrl+C/Cmd+C).
+ 
+The **Share** Button is the go-to to share the log opened in side panel to other contexts. 
+
+* **Copy the JSON log** content to the clipboard (or use keyboard shortcut interaction (Ctrl+C/Cmd+C)).
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
* **Copy to clipboard** or `Ctrl+C` / `Cmd+C` copies the log JSON to your clipboard.
```

File: content/en/logs/explorer/_index.md [hunk 22]
```diff
@@ -104,7 +141,11 @@ Interact with the upper reserved attributes section:
...
+* **Share a link** to the log (along with the underlying view) to teammates through email, Slack and more (see our [notification integrations][5]) with Share Event interaction.
```
Comment (ruthnaebeck) [inline content/en/logs/explorer/_index.md]:
```suggestion
* **Share Event** shares the log (along with the underlying view) with teammates through Email, Slack, and more. See all [Datadog notification integrations][4] available. 
```
Link 5 doesn't exist for this tab. The next number is 4. Add:
`[4]: /integrations/#cat-notification` at line 156

File: content/en/security/logs.md [hunk 23]
```diff
@@ -52,6 +52,7 @@ Additionally, certain features are not available at the moment to customers who
 * The logs Live Tail, Rehydrate from Archives, and Generate Metrics features are disabled
 * Notifications from Log Monitors cannot include log samples
 * Log Monitors cannot be configured with a `group-by` clause
+* You cannot [share][6] logs (nor traces) from the explorer through web integrations
```
Comment (ruthnaebeck) [inline content/en/security/logs.md]:
```suggestion
* You cannot [share][6] logs (nor traces) from the explorer through web integrations.
```

File: content/en/security/logs.md [hunk 24]
```diff
@@ -64,3 +65,4 @@ If you have any questions about how the Log Management Service satisfies the app
 [3]: /agent/logs/advanced_log_collection/?tab=exclude_at_match#filter-logs
 [4]: /agent/logs/advanced_log_collection/#scrub-sensitive-data-from-your-logs
 [5]: /integrations/amazon_lambda/#log-collection
+[6]: /logs/explorer/#share-view
```
Comment (ruthnaebeck) [inline content/en/security/logs.md]:
```suggestion
[6]: /logs/explorer/#share-views
```

**Pull request merged**